wraith/dark.yaml

```diff
@@ -0,0 +1,44 @@
+imports: "core.yaml"
+
+# The directory that your base screenshots will be stored in.
+history_dir: 'wraith/then/dark'
+
+# The directory that your latest screenshots will be stored in
+directory: 'wraith/now/dark'
+
+paths:
+  home:
```

Comment:
why aren't we using the paths from core?

Reply:
@Throne3d's idea, to cut down the number of files we're keeping an stuff -- all the themes except iconless (and logged out I guess technically) that override that have five or less users.